src/hooks/useCheckClientName.ts

```diff
@@ -0,0 +1,10 @@
+import type CozyClient from 'cozy-client'
+
+import { useEffect } from 'react'
+
+import { checkClientName } from '/app/domain/authentication/services/SynchronizeService'
+import { safePromise } from '/utils/safePromise'
+
+export const useCheckClientName = (client: CozyClient): void => {
+  useEffect(() => safePromise(checkClientName)(client), [client])
```

Comment:
Should we use `useClient` here instead of passing it in parameters?

Reply:
yes I think you're right it would look more streamlined

- [ ]